qa-covid19.planx-pla.net/manifest.json

```diff
@@ -4,33 +4,33 @@
     "That's all I have to say"
   ],
   "versions": {
-    "arborist": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/arborist:2021.04",
-    "augur": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/gen3-augur:fix_docker2",
-    "awshelper": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/awshelper:2021.04",
+    "arborist": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/arborist:2021.07",
+    "augur": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/gen3-augur:2021.07",
+    "awshelper": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/awshelper:2021.07",
     "aws-es-proxy": "quay.io/cdis/aws-es-proxy:0.8",
-    "covid19-etl": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/covid19-etl:4.8.2",
-    "covid19-notebook-etl": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/covid19-notebook-etl:4.7.6",
-    "covid19-bayes-model": "quay.io/cdis/covid19-bayes-model:3.4.3",
-    "fence": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/fence:4.29.1",
-    "indexd": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/indexd:2021.04",
-    "peregrine": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/peregrine:2021.04",
-    "pidgin": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/pidgin:2021.04",
-    "revproxy": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/nginx:2021.04",
-    "sheepdog": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/sheepdog:2021.04",
-    "portal": "quay.io/cdis/data-portal:feat_WLSmapDataOverlays",
-    "tube": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/tube:0.5.0",
+    "covid19-etl": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/covid19-etl:2021.07",
+    "covid19-notebook-etl": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/covid19-notebook-etl:2021.07",
+    "covid19-bayes-model": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/covid19-bayes-model:2021.07",
```

Comment:
these are not included in the monthly release

Reply:
shouldn't they be included?

Reply:
no, updates to the PRC ETLs are usually small and urgent to deploy, there never are changes on master waiting to be deployed

Reply:
I have added them to the ignore list for jenkins job. So next time when the job is run, the job wouldnt replace the version of these services @paulineribeyre